test/chart/generateCategoricalChart.spec.tsx

```diff
@@ -230,4 +231,27 @@ describe('generateCategoricalChart', () => {
       );
     });
   });
+
+  describe('createDefaultState', () => {
+    it('Should have the correct dataIndex', () => {
```

Comment:
Why did you combine these four separate test cases into a single test? Is there any interaction that I am missing? If not, please split it up into 4 tests.

Reply:
it was just ctrl+c,v :sweat_smile: